app/controllers/users_controller.rb

```diff
@@ -12,18 +13,30 @@ class UsersController < ApplicationController
   def index
     @user = current_user
     @gds = current_user.permissions.admin_management
+    @teams = Team.all
+    @relying_parties, @identity_providers, @other = @teams.group_by(&:team_type).values_at(TEAMS::RP, TEAMS::IDP, TEAMS::OTHER)
+    @all_rps_and_idps = Team.all
```

Comment:
You can replace 18 & 19 with `@all_rps_and_idps = @teams - @other unless @other.nil?`